src/lib/stitching/vortex/stitching.ts

```diff
@@ -342,6 +356,9 @@ export const vortexStitchingEnvironment = (localSchema: GraphQLSchema) => ({
               response.__typename = removeVortexPrefix(typename)
               return response
             })
+            .catch(() => {
+              return null
+            })
```

Comment:
Added lines 359-361 above to add some protection in case we are trying to query for viewing rooms that were recently deleted.  I am a little not confident in this as a solution - what do yall think? One thing that is tricky for me is that we are fetching objects from different places (viewing room data is coming from GravityQL, shows/artist/artwork data is coming in from Gravity data loaders. 
Looking like queries that hit the gravity loaders are catching errors in case of 404s at the loader level.


Here's the experience WITHOUT this catch when trying to fetch analytics data for a viewing room that was recently deleted:

<img width="1648" alt="Screen Shot 2021-01-15 at 10 19 06 AM" src="https://user-images.githubusercontent.com/12748344/104752569-d477eb00-5724-11eb-9681-092b69118fb3.png">

Here is the response with the catch added here. We return `null` for the entity object if the ViewingRoom cannot be fetched from GravityQL. Looking like Volt is prepared to handle this pattern already. (show viewing analytics data for deleted shows for example.

<img width="1656" alt="Screen Shot 2021-01-15 at 10 21 35 AM" src="https://user-images.githubusercontent.com/12748344/104752990-61bb3f80-5725-11eb-90d4-c909d4b7a174.png">


TL;DR I don't want to an error to bubble up to Sentry if a VR is missing! Would prefer we just return null for the object and continue on our way.  Would love to hear your feedback here!

Reply:
Discussed with the crew - since this is existing behavior we will resolve the root cause of this exception as part of a separate ticket: https://artsyproduct.atlassian.net/browse/PX-3637